app/src/main/kotlin/dev/msfjarvis/aps/db/entity/StoreEntity.kt

```diff
@@ -18,9 +22,15 @@ data class StoreEntity(
   @ColumnInfo(name = "name")
   val name: String,
 
+  @ColumnInfo(name = "uri")
```

Comment:
`name` value looks redundant in most fields here since it matches the field name.

Reply:
It's just to make the code look consistent.